nohost/src/NoHostServer.js

```diff
@@ -42,9 +43,22 @@ define(function (require, exports, module) {
      * instrumentation is enabled
      */
     NoHostServer.prototype.add = function (liveDocument) {
+        var id = liveDocument.doc.language.getId();
+
         if (liveDocument.setInstrumentationEnabled) {
-            // enable instrumentation
-            liveDocument.setInstrumentationEnabled(true);
+            //If new HTML document is empty, inject defaultHTML file
+            //so live preview doesn't break
+            if(liveDocument.doc.getText() === "" && id === "html") {
+                liveDocument.doc.setText(defaultHTML);
```

Comment:
Okay, so we need a separate HTML5 compliant `html` shell page that is loaded whenever a new html file is created. Can you make one and require it in place of `defaultHTML`?